scripts/ci/run_perftest.sh

```diff
@@ -116,14 +125,6 @@ go get \
   github.com/influxdata/influxdb-comparisons/cmd/bulk_query_gen \
   github.com/influxdata/influxdb-comparisons/cmd/query_benchmarker_influxdb
 
-# hack to get the daemon to start up again until https://github.com/influxdata/influxdb/issues/21757 is resolved
-systemctl stop influxdb
```

Comment:
On lines 118/119, do we even need to clone the comparisons repo?

Reply:
I don't think so - nice!

- [ ] Try removing the 'git clone' above for influxdb-comparisons